pkg/services/accesscontrol/roles.go

```diff
@@ -0,0 +1,115 @@
+package accesscontrol
+
+import "github.com/grafana/grafana/pkg/models"
+
+var PredefinedRoles = map[string]RoleDTO{
```

Comment:
Are we going to rename the database table to `predefined_roles` and reflect that in the code if relevant? In other words, should talk about "predefined roles" and not "built-in roles" from now on?

Reply:
No, unfortunately those are two slightly different concept, the built-in roles are the Viewer/Editor/Admin trio (plus Grafana Admin) and predefined roles are just a list of roles that _we_ define. It'd be nice with a better/clearer naming here, but unfortunately, Viewer/Editor/Admin are already called "role" in a lot of places.

We could rename the database table to something with "org role" maybe, because it's really what role the user has access to organization wide?